src/main/java/com/redhat/rhjmc/containerjfr/platform/internal/DefaultPlatformClient.java

```diff
@@ -67,7 +67,7 @@
                         u -> {
                             try {
                                 return new ServiceRef(
-                                        u.getJmxServiceUrl().toString(), u.getMainClass(), 0);
+                                        u.getJmxServiceUrl().toString(), u.getMainClass());
```

Comment:
If the `ServiceRef` contains a service URL then the `toString` here can be removed